cumulusci/cli/cci.py

```diff
@@ -272,6 +279,17 @@ def connection_error_message():
     click.echo(click.style(message, fg="red"), err=True)
 
 
+def yaml_parse_error_message(error):
+    message = f"An error occurred while parsing a yaml file (likely cumulusci.yml). Check around line {error.problem_mark.line} for the issue."
```

Comment:
I think the cci_safe_load function in cumulusci.utils.yaml.cumulusci_yml would be a better place to catch parse errors and reraise them with a better message -- then we should be able to get the filename. We should also include the column if it's available, and include the original error message for those who can interpret it.